src/main/java/de/doubleslash/usb_led_matrix/view/ConfigurationView.java

```diff
@@ -430,9 +432,23 @@ void refreshButton() {
       }
    }
 
-   void turnOffAutomaticallyIfNeeded(final LocalTime currentTime) {
+   void turnOffAutomaticallyIfNeeded(final LocalTime currentTime, LocalDate startDate) {
       final LocalTime timeLightsTurnedOff = timeLightsTurnedOffNow.plusMinutes(CommandLineOptions.getTimeout());
-      if (!currentTime.isBefore(timeLightsTurnedOff) && !timedOut) {
+
+      LocalDate currentDate = LocalDate.now();
+      if (!currentDate.isEqual(startDate)) {
+         timedOut = false;
+         startDate = currentDate;
+      }
+
+      LocalDate turnOffDate = startDate;
+      if (timeLightsTurnedOff.isBefore(timeLightsTurnedOffNow)) {
+         turnOffDate = turnOffDate.plusDays(1);
+      }
+
+      LocalDateTime turnOffDateTime = LocalDateTime.of(turnOffDate, timeLightsTurnedOff);
```

Comment:
can't we use LocalDateTime from the start? so that we only have LocalDateTime and not one localDate and one LocalTime to represent a date and a time?